examples/step-2/doc/results.dox

```diff
@@ -76,3 +50,31 @@ i.e. the matrix clusters more around the diagonal.
 Another idea of experiments would be to try other renumbering
 strategies than Cuthill-McKee from the DoFRenumbering namespace and see how
 they affect the sparsity pattern.
+
+You can also visualize the output using <a
+href="http://www.gnuplot.info/">GNUPLOT</a> (one of the simpler visualization
+programs; maybe not the easiest to use since it is command line driven, but
+also universally available on all Linux and other Unix-like systems) by changing from "print_svg()" to "print_gnuplot" in distribute_dofs() and renumber_dofs():
```

Comment:
mark up all of the calls using `<code>...</code>` (and no quotes). Be consistent about using parentheses after the function name.